bosh-stemcell/lib/bosh/stemcell/stage_collection.rb

```diff
@@ -36,7 +36,7 @@ def agent_stages
         :bosh_micro_go,
         :aws_cli,
         :logrotate_config,
-      ]
+      ].reject{ |s| RbConfig::CONFIG['host_cpu'] == 'powerpc64le' and s ==  :bosh_micro_go }
```

Comment:
You mean we don't need to patch packages like ruby and yaml, sounds good. I thought it may be required for smth else.